DEV_GUIDE.md

```diff
@@ -221,13 +240,13 @@ have done a recent build, for example:
 Then you can run the individual test using the `-Dit.test` option, for example:
 
 ```sh
-mvn -pl testing -Pit-cassandra-3.11 verify -Dit.test=RestApiv2SchemaTest
+mvn -pl testing -Pit-cassandra-4.0 verify -Dit.test=RestApiv2SchemaTest
```

Comment:
This will soon stop working due to refactoring (could be changed/removed as follow up?).

Reply:
replacing with reference to one of the CQL integration tests (also on the next command)